wapitiCore/attack/mod_http_headers.py

```diff
@@ -0,0 +1,78 @@
+import requests
+from wapitiCore.attack.attack import Attack
+from wapitiCore.net.web import Request
+from wapitiCore.language.vulnerability import Additional
+#This module check the recommendations of security parameters in HTTP headers
```

Comment:
What about parsing the `Content-Security-Policy` header ?

As it is more tricky we may consider adding a dependancy but I think it would worth it

Reply:
i'm working on a new module who check CSP .. i'll push it once i finish it